bps/templates/README.md

```diff
@@ -0,0 +1,61 @@
+BPS Template Files for Calibration Construction
+===============================================
+
+This directory contains BPS template files for calibration testing and
+construction. These are meant to conform to the naming conventions laid out in
+[DMTN-222](https://dmtn-222.lsst.io).
+
+Overview
+--------
+
+These template files are only part of calibration construction. They are meant
+to be flexible enough to use for both test construction into a user collection,
+as well as final construction into the overall collection.
+
+The templates rely on a lot of exported environment variables, defined
+below. Note that because the bps file does not share an env with the caller all
+the env vars must be explicitly exported.
+
+The calibrations also assume that the caller is continuously certifying
+calibrations for the next step into a collection
+`${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}/${TAG}`.
+
+Environment Variables
+---------------------
+
+These are the environment variables that must be used. Examples are substituted in.
```

Comment:
This should also note that `selection_string` is a dummy variable here, and that it needs to be filled with something explicit.  I was going to suggest putting in the information being used on DM-46562 (the ticket used below), but I see that's this ticket adding the BPS templates, not a generation ticket.